python/src/scipp/plotting/model.py

```diff
@@ -125,62 +143,48 @@ def _axis_coord_and_formatter(self, dim, data_array, dim_to_shape,
 
         # Set up tick formatters
         if dim in dim_label_map:
-
-            if data_array.meta[
-                    dim_label_map[dim]].dtype == sc.dtype.vector_3_float64:
-                # If the non-dimension coordinate contains vectors
-                form = self._vector_tick_formatter(
-                    data_array.meta[dim_label_map[dim]].values,
-                    dim_to_shape[dim])
-                formatter.update({"custom_locator": True})
-            elif data_array.meta[dim_label_map[dim]].dtype == sc.dtype.string:
-                # If the non-dimension coordinate contains strings
-                form = self._string_tick_formatter(
-                    data_array.meta[dim_label_map[dim]].values,
-                    dim_to_shape[dim])
-                formatter.update({"custom_locator": True})
+            key = dim_label_map[dim]
+        else:
+            key = dim
+
+        if data_array.meta[key].dtype == sc.dtype.vector_3_float64:
+            # If the non-dimension coordinate contains vectors
```

Comment:
Those comments are no longer correct, this code is for non-dim and dim coordinates.